osu.Game/Screens/Play/BeatmapMetadataDisplay.cs

```diff
@@ -49,6 +49,44 @@ public MetadataLine(string left, string right)
             }
         }
 
+        private class EpilepsyWarning : FillFlowContainer
+        {
+            public EpilepsyWarning(bool warning)
+            {
+                if (warning)
+                    this.Show();
+                else
+                    this.Hide();
+
+                AutoSizeAxes = Axes.Both;
+                Direction = FillDirection.Vertical;
+                Children = new Drawable[]
+                {
+                    new SpriteIcon
+                    {
+                        Anchor = Anchor.TopCentre,
+                        Origin = Anchor.TopCentre,
+                        Icon = FontAwesome.Solid.ExclamationTriangle,
+                        Size = new Vector2(40),
+                    },
```

Comment:
UX-wise this warning doesn't really... warn that much. It blends in too much with the rest of the metadata.

I'd say it needs to be jump out more; the simple way would be applying some kind of colour to the icon, for instance:

![2020-07-21-234918_619x134_scrot](https://user-images.githubusercontent.com/20418176/88110748-01e83f80-cbad-11ea-83c6-af9aca99588a.png)

Not sure whether any further animations should be added (thought about flashing the icon from white to yellow for a couple of seconds, but on second thought flashing an icon in an *epilepsy warning* seems... not wise...)

Reply:
for the UX, In my opinion, the epilepsy warning should not be embedded with the beatmap metadata display, it should rather be displayed on the center of the screen, the track volume should get lowered and background gets dimmed, all to get proper and serious attention from the user; I've checked out the branch and made some changes to get to the idea above, here is what I got: https://streamable.com/uevzep

Reply:
Looks ok but the transition away needs to be longer. I could barely read the text in time on that video.